airflow/www/templates/airflow/dag_audit_log.html

```diff
@@ -0,0 +1,92 @@
+{#
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+#}
+
+{% extends "airflow/dag.html" %}
+{% block title %}Dag Audit Log{% endblock %}
+
+{% block head_css %}
+{{ super() }}
+<link href="{{ url_for_asset('dataTables.bootstrap.min.css') }}" rel="stylesheet" type="text/css" >
+<link href="{{ url_for_asset('bootstrap-toggle.min.css') }}" rel="stylesheet" type="text/css">
+{% endblock %}
+
+{% block content %}
+  {{ super() }}
+  <h2>Dag Audit Log</h2>
+    <h4 style="display: block; padding-top: 10px; padding-bottom: 4px">
+      <p>
+        Selected events and operations in the last {{ max_query_days }} days.
+        For full list of events click
+        <a href="{{ url_for('LogModelView.list', _flt_3_dag_id=dag_id) }}">here</a>.
+      </p>
+      <p>User operations with comments are <span style="color: #158CBA">highlighted.</span></p>
+    </h4>
+    <table id="dag_audit_log_table" class="table table-striped table-bordered table-hover">
+        <thead>
+            <tr>
+                <th style="min-width: 140px">Time</th>
+                <th style="min-width: 80px">Task Id</th>
+                <th>Event</th>
+                <th style="min-width: 220px">Execution Date</th>
+                <th>By User</th>
```

Comment:
```suggestion
                <th>Owner</th>
```

I would be in favour of changing this column title to `Owner` to better align with the column in the DB and the column name in the `Audit Logs` view, however I agree that `User` is a more accurate title in most cases. 

Thoughts?